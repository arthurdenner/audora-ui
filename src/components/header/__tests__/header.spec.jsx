import React from 'react';
import { render, mount } from 'enzyme';
import Header from '../header';

describe('Header', () => {
  it('renders correctly', () => {
    const wrapper = render(<Header full={false}><p>header</p></Header>);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly when full', () => {
    const wrapper = mount(<Header full><p>header</p></Header>);
    expect(wrapper.props().full).toBe(true);
  });

  it('renders correctly when fixed', () => {
    const wrapper = mount(<Header fixed><p>header</p></Header>);
    expect(wrapper.props().fixed).toBe(true);
  });
});
