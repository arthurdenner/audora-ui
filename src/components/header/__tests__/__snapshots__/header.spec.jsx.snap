// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  class="sc-bdVaJa iZIqFp"
>
  <header
    class="sc-bwzfXH jlNGUd"
  >
    <p>
      header
    </p>
  </header>
</div>
`;
